web/models.py

```
@@ -2285,20 +2338,11 @@ def status(self):
 
         # Check if there's a passing score defined on the quiz
         passing_score = getattr(self.quiz, "passing_score", 0)
-        if passing_score and self.score >= passing_score:
+        if self.calculate_score() >= passing_score:
             return "passed"
         else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)